concordium-contracts-common/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Set minimum Rust version to 1.65.
 - Implement `serde::Serialize` and `serde::Deserialize` for `Duration` using `FromStr` and `Display` implementations, when feature `derive-serde` is enabled.
+- Add `smart-contract` feature to enable the macros which are only suitable for smart contract development.
```

Comment:
I think we need to make it clear that means using concordium-std. "smart contract development" is quite ambiguous.

Reply:
Make it clear it is not enabled by default.